libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -699,6 +699,11 @@ AP_InertialSensor::detect_backends(void)
         ADD_BACKEND(AP_InertialSensor_HIL::detect(*this));
         return;
     }
+
+    if (_sitl_mode) {
```

Comment:
why is this needed?

Reply:
This is to prevent AP_InertialSensor from reporting poor health when in SITL with 0 accelerometer offsets. Per your comment below, looks like this approach throws out the baby with the water though...